skymp5-server/cpp/unit/TestUtils.hpp

```diff
@@ -8,41 +8,19 @@
 #include <nlohmann/json.hpp>
 #include <thread>
 
-using namespace Catch;
-
 // Utilities for testing
-namespace {
-std::string MakeMessage(const nlohmann::json& j)
-{
-  std::string s;
-  s += (char)Networking::MinPacketId;
-  s += j.dump();
-  return s;
-}
+namespace TestUtils{
```

Comment:
Please run formatting (can be done in Visual Studio or by manually running clang-format)

Reply:
a bit of discussion purely, not yet a proposal to change something:

Also unsure if there are namespaces named in `PascalCase` in the project. And probably there's even no reason to put that into a namespace.